### PR TITLE
adding cronus simulator template

### PR DESCRIPTION
**Pull request by dhalimi**
Adding new template under openstack/cronus for cronus-simulator

### CODE REVIEW

File: openstack/cronus/values.yaml [hunk 1]
```diff
@@ -187,3 +188,30 @@ ingress-nginx:
...
+    podManagementPolicy: OrderedReady
+    annotations: {}
+    labels: {}
+    headless:
+      annotations: {}
+      labels: {}
+      servicePort: 80
+  updateStrategy: RollingUpdate
+  replicas: 1
+  nebula:
```
Comment (kayrus) [inline openstack/cronus/values.yaml:210]: what should be under the nebula? Empty string?

File: openstack/cronus/values.yaml [hunk 2]
```diff
@@ -187,3 +188,30 @@ ingress-nginx:
...
+
+simulator:
+  enabled: true
+  image:
+    name: cronus-exporter-repository
+    pullPolicy: IfNotPresent
+    tag: latest
+    imagePullSecrets: []
+  statefulSet:
+    enabled: true
```
Comment (kayrus) [inline openstack/cronus/values.yaml:200]: I think there is no need to use statefulsets here. You don't use locks or anything else.

File: openstack/cronus/values.yaml (hunk 1 above)
Comment (dhalimi) [reply openstack/cronus/values.yaml:210]: Yes, it gives the option to use a local nebula for debugging

File: openstack/cronus/values.yaml (hunk 2 above)
Comment (dhalimi) [reply openstack/cronus/values.yaml:200]: I've disabled in secrets

**Comment by kayrus** [comment]
LGTM

**Review by kayrus — APPROVED** [review]

**Pull request merged**